lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1343,8 +1343,6 @@ static llvm::Expected<ParsedExpression> ParseAndImport(
       sc.target_sp->GetSwiftEnableBareSlashRegex();
   invocation.getLangOptions().EnableBareSlashRegexLiterals =
```

Comment:
The test cases you've added use bare slash literals, seems that we can remove this too?

Reply:
I might be misunderstanding, but `#/.../#` is not a bare slash literal. We still need to have the `EnableBareSlashRegexLiterals` option to match the compiler's `--enable-bare-slash-regex` option, which is disabled by default (Xcode just happens to pass it by default). We could enable it by default in LLDB if we wanted, though we'd still want the option in case the user wants to disable it.

Reply:
I misinterpreted the tests, thanks for clarifying.